### DISCUSSION TITLE
Dashboard cleanup

### DISCUSSION THREAD

**Discussion by motionthings** [getgrav/grav#3795] [General]
After using Worpress for 15 years I have switched to Grav.
I love the idea of a flat file CMS that don't need a database backend and that supports heavy caching out of the box.

It seems to be getting regular updates too.

But.
The 'Notifications' and 'Newsfeed' section in the dashboard _really_ makes this project look "stale".

I get the point that you can connect to the site with Code, Sublime, Vim or whatever text editor and edit the files directly in that, but that's not something I can present to potential clients or beginners.
They are used to managing their sites in their browser.

Could someone put release notes in the 'Notifications' and maybe a "site of the week" (or month) in the 'Newsfeed' section?
That way it looks like more things are happening?

Just a little suggestion from me.
Keep up the great work :)

**Comment by rhukster** [comment]
Good suggestions, and definitely somethign I want to get around to.
